File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/user_coupon_list_form.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class UserCouponListForm:
    """用户优惠券列表查询请求"""
    status: str = None
    type: str = None
    page_num: int = None
    page_size: int = None

File: sdkwork-app-sdk-typescript/src/types/user-coupon-list-form.ts
/** 用户优惠券列表查询请求 */
export interface UserCouponListForm {
  /** 状态: UNUSED-未使用, USED-已使用, EXPIRED-已过期 */
  status?: string;
  /** 优惠券类型 */
  type?: string;
  /** 页码 */
  pageNum?: number;
  /** 每页大小 */
  pageSize?: number;
}
